advanced/index.md

```diff
@@ -71,15 +71,15 @@ Historical commit points are simple to discuss using *commitish* rather than hex
 Shorthand              | Explanation
 ---------------------- | -----------
 `HEAD`                 | Current commit
-`HEAD^^`               | Second parent of current commit
-`HEAD~2`               | Two commits from current commit
+`HEAD^2`               | Second parent of current commit (assumes current commit the result of a merge)
+`HEAD~2`               | First parent (commit) of the first parent of the current commit (different than above!)
```

Comment:
Yeah, I wasn't totally happy with this either. First and second describe both what parent one's discussing for a single merge commit (i.e. first is the branch you were on, second is the branch you merged in). But of course they're also used to say how far back in history one goes. Totally open to suggestions.